spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/ModuleDefinitionBuilder.java

```diff
@@ -108,4 +117,14 @@ else if (argument.getName().equalsIgnoreCase("outputType")) {
 		}
 		return moduleDefinitions;
 	}
+
+	private String removeChannelTypePrefix(String sourceChannelName) {
```

Comment:
Can we change the parameter name `sourceChannelName` to just `channelName` as it applies for both source/sink named channel names?